cli/lsp/testing/execution.rs

```diff
@@ -270,8 +270,11 @@ impl TestRun {
     let tests: Arc<RwLock<IndexMap<usize, test::TestDescription>>> =
       Arc::new(RwLock::new(IndexMap::new()));
     let mut test_steps = IndexMap::new();
-    let worker_factory =
-      Arc::new(factory.create_cli_main_worker_factory().await?);
+    let worker_factory = Arc::new(
+      factory
+        .create_cli_main_worker_factory(Default::default())
```

Comment:
Now this is a viral change to the signature - can we add a specialized API that accepts an argument so we don't have to do this `Default::default()` all over the place?